.flake8

```diff
@@ -35,13 +35,16 @@ exclude =
     cfgov/jobmanager/migrations/*.py,
     cfgov/v1/migrations/*.py,
     cfgov/ask_cfpb/migrations/*.py,
+    cfgov/selfregistration/migrations/*.py,
+    cfgov/agreements/migrations/*.py,
```

Comment:
While you are editing this file, how would you feel about collapsing this list into the wildcard `*/migrations/*.py`, as suggested in @willbarton's excellent template [here](https://github.com/cfpb/development/pull/154/files#diff-540ed609f086a01ee6f84e1a7bba5f6eR16)?

Reply:
That's a good idea

Reply:
done!